src/main/java/edu/mit/dos/object/FileService.java

```diff
@@ -39,36 +39,24 @@
     private ServiceConfig serviceConfig;
 
     /**
-     * Retrieves files by oid
+     * Retrieves files by fid
      *
-     * @param oid
+     * @param fid
```

Comment:
For future maintenance, it might be useful to briefly add a description of what fid is.